docs/how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -22,7 +22,7 @@ This document requires the presence of the ACRN hypervisor and Kata Containers o
 
 - ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
   > **Note:** Please make sure to have a minimum of 4 logical processors (HT) or cores.
-- ACRN [software](https://projectacrn.github.io/latest/tutorials/kbl-nuc-sdc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- ACRN [software](https://projectacrn.github.io/2.1/tutorials/kbl-nuc-sdc.html#use-the-script-to-set-up-acrn-automatically) setup.
```

Comment:
@gvancuts, do you happen to know why this got removed from the `latest` and what would be a good replacement for that URL?

@Bevisy, I think we should not point to the `2.1` repo, but instead check whether something has to be updated from our side, and Geoffrey may be the best person to guide us through the needed changes.

Reply:
cc @vijaydhanraj Do you know why those doc pages were removed for releases following 2.1? Can you suggest what would be the appropriate doc to link to here ?